modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/ButtonDropDown/index.js

```diff
@@ -39,39 +44,116 @@ const ButtonDropDown = ({
 			}
 			{...props}
 		>
-			<ClayDropDown.ItemList>
-				{items.map(
-					({
-						customOptionStyle,
-						disabled,
-						icon,
-						label,
-						onClick,
-						tooltip,
-					}) => (
-						<ClayDropDown.Item
-							className={classNames(
-								'font-weight-semi-bold text-paragraph-sm px-3 rounded-xs',
+			{!isAdminOrPartnerManager && label === 'Actions' && (
+				<ClayDropDown.ItemList>
+					{filteredDropdownItems.map(
+						({
+							customOptionStyle,
+							disabled,
+							icon,
+							label,
+							onClick,
+							tooltip,
+						}) => (
+							<ClayDropDown.Item
+								className={classNames(
+									'font-weight-semi-bold text-paragraph-sm px-3 rounded-xs',
+									customOptionStyle,
+									{
+										'cp-common-drop-down-item text-neutral-8': !disabled,
+										'text-neutral-5': disabled,
+									}
+								)}
+								disabled={disabled}
+								key={label}
+								onClick={onClick}
+								title={disabled ? tooltip : null}
+							>
+								<div className="d-flex">
+									{icon && <div className="mr-1">{icon}</div>}
+
+									{label}
+								</div>
+							</ClayDropDown.Item>
+						)
+					)}
+				</ClayDropDown.ItemList>
+			)}
+
+			{isAdminOrPartnerManager ||
```

Comment:
Se a label download aparece tanto pra admins quanto pra não admins, essa lógica é necessária?

Reply:
Para os **não admin/partnerManager**, eu preciso especificar quando o botão é o "download" e quando é "actions", pois o map é feito de forma distinta em cada caso. Quando o botão for Actions, o map é feito na variável filtrada. Quando o botão for download, o map é feito na variável normal (items).